static/lang/re-en.json

```diff
@@ -3036,6 +3036,122 @@
                 "Tenth": "10th-Rank",
                 "Third": "3rd-Rank"
             },
+            "SpellTrickster": {
+                "AgileHand": {
+                    "Label": "Agile Hand",
```

Comment:
We shouldn't need the labels for these since the rules are on the items

Reply:
The labels are for the suboptions in Spell Trickster Dedication. Or do you mean I should just use `{item|description}` the body of each Item Alteration instead of making localisation keys?

Reply:
If we get dynamic suboptions then no need for the label at least. The notes can stay. But let's hold here for a bit and put this in draft.